src/Insurello.AsyncExtra/AsyncExtra.fs

```diff
@@ -44,10 +44,21 @@ module AsyncResult =
 
                 | Error err -> Async.singleton (Error err))
 
+    let bindError: ('errT -> AsyncResult<'x, 'errU>) -> AsyncResult<'x, 'errT> -> AsyncResult<'x, 'errU> =
+        fun f asyncResultErrT ->
+            asyncResultErrT
```

Comment:
As it is the last param you don't have to name it and can just return `Async.bind`